src/components/widgets/buttons/ResourceButton.js

```diff
@@ -12,7 +14,22 @@ const fileSelectedIcon = require("../../../images/icons/fileSelected.png");
 const presqtMetadataFileIcon = require("../../../images/icons/presqtMetadataFile.png");
 const presqtMetadataFileIconSelected = require("../../../images/icons/presqtMetadataFileSelected.png");
 
-export default function ResourceButton({browserType, resource, level, onClick}) {
+export default function ResourceButton({ browserType, resource, level, onClick }) {
+  // TRANSFER SPECIFIC CODE
```

Comment:
Maybe this should be its own component?

Reply:
Ehhh I remember all the rest of the code in this button...maybe not haha Let's talk about it